bridge/src/test/java/com/bitwarden/bridge/util/EncryptionUtilTest.kt

```diff
@@ -0,0 +1,178 @@
+package com.bitwarden.bridge.util
+
+import com.bitwarden.bridge.model.AddTotpLoginItemData
+import com.bitwarden.bridge.model.SharedAccountData
+import com.bitwarden.bridge.model.SymmetricEncryptionKeyData
+import com.bitwarden.bridge.model.toByteArrayContainer
+import io.mockk.every
+import io.mockk.mockkStatic
+import io.mockk.unmockkStatic
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.Assertions.assertNotNull
+import org.junit.jupiter.api.Assertions.assertTrue
+import org.junit.jupiter.api.Test
+import java.security.MessageDigest
+import java.security.NoSuchAlgorithmException
+import java.time.Instant
+import javax.crypto.Cipher
+import javax.crypto.KeyGenerator
+
+class EncryptionUtilTest {
+
+    private val TEST_SHARED_ACCOUNT_DATA = SharedAccountData(
```

Comment:
Do we typically append TEST to the name of sample data instances? Also, I think we tend to place these at the bottom of a test file.

Reply:
I swear I saw `TEST_` prefix elsewhere, but I must be seeing ghosts. Agreed on file location too. Updated:

0d465a81ae09aba1ae5216932b976624bfaefcbd

This also explains why the IDE was complaining to me about the name formatting of these... it is no longer complaining!

Reply:
I see the change in the commit you posted but in the full diff this has not changed....maybe that commit was reverted in a later one?

Reply:
I must have made an error while rebasing... thank you for catching this!!

Here's an update:

af046f9ff733328ac565300f1dca6eeae0054b41

Reply:
Yeah I messed something up rebasing this morning for sure... here's the missing commit:

af046f9ff733328ac565300f1dca6eeae0054b41